experimental/packages/opentelemetry-instrumentation-http/test/functionals/http-enable.test.ts

```diff
@@ -1598,4 +1598,111 @@ describe('HttpInstrumentation', () => {
       );
     });
   });
+  describe('URL Redaction', () => {
```

Comment:
Would it make sense to add a test to ensure that URLs without redaction needed aren't affected?